### PR TITLE
CI jobs for memory leaks and corruptin

### PR DESCRIPTION
**Pull request by mnunberg**
@rafie fyi

### CODE REVIEW

File: .circleci/ci_get_deps.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -x
+./srcutil/get_gtest.sh
+python ./src/pytest/test_rdb_compatibility.py
+
+pip install wheel
+pip install setuptools --upgrade
+pip install git+https://github.com/RedisLabsModules/RLTest.git@concurrent
```
Comment (MeirShpilraien) [inline .circleci/ci_get_deps.sh]: Please change to master as we already merge it

File: src/pytest/test_rdb_compatibility.py [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
-        os.makedirs(REDISEARCH_CACHE_DIR)
-    
-    for fileName in filesNames:
+    rdbFilePath = os.path.join(dbDir, dbFileName)
+    if not downloadFiles():
+        if os.environ.get('CI'):
+            env.assertTrue(False)  ## we could not download rdbs and we are running on CI, let fail the test
+        else:
+            env.skip()
+            env.stop()
```
Comment (MeirShpilraien) [inline src/pytest/test_rdb_compatibility.py]: Why you stop the env here?

**Comment by MeirShpilraien** [comment]
@mnunberg the check-before-flush PR is merge, you can move to master.

**Comment by mnunberg** [comment]
@MeirShpilraien we are using master now

**Pull request merged**